src/jobs/pages/erfaren-skyutvikler-i-trondheim.md

```diff
@@ -0,0 +1,71 @@
+
+<div class="left"><img alt="Sarah hos Variant holder et foredrag" src="/images/utvikler-sarah.png"/></div>
+
+## Våre forventninger
+
+Hos oss vil du jobbe med ulike arbeidsoppgaver ut i fra egne ønsker om bransje og teknologi. Vi vet ikke nøyaktig hvilke oppdrag vi får. Det er faktisk slik at din kompetanse og dine ønsker ofte styrer det neste oppdraget. Viktigste for oss er at du har et ønske om å være faglig dyktig.
+
+Som erfaren konsulent vil man ofte forventes å ta litt ansvar for både tekniske valg og utviklingsprosessen i et team. Vi lever i skjæringspunktet mellom design, utvikling og mennesker, og derfor tror vi det er en fordel om du har jobbet i kryssfunksjonelle team før. Det er også en fordel om du har jobbet i konsulentbransjen, og kan se muligheter, behov og løsninger hos våre kunder. Om det gjelder teknologi, innovasjon eller produktutvikling.
+
+## Lønn og kompensasjoner
+
+Vi tilstreber åpenhet i alt vårt arbeid. Derfor forsøker vi å være forutsigbar med tanke på de ulike insentivene vi tilbyr. Totalt består dette av fastlønn og bonus:
+
+- Variant justerer lønn basert på Teknas lønsstatistikk. Dette betyr at hvis du eksempelvis har en [mastergrad fra 2013, vil din startlønn være 808 168](/kalkulator?year=2013&degree=masters).
+- Du vil sammen med alle ansatte dele 30% av overskuddet pr kvartal som en resultatbonus.
+- Gjennom Medeierskapsprogrammet i Variant [tilbys alle ansatte å kjøpe aksjer](https://blog.variant.no/invitasjon-til-%C3%A5-kj%C3%B8pe-aksjer-i-variant-as-27a29a307cb2) i Variant. Første april hvert år selger selskapet aksjer i selskapet til alle ansatte.
+- Vi har en [håndbok](https://handbook.variant.no/) for hva du enn måtte lure på. (Pro tip: sjekk ut [Andre goder og ytelser](https://handbook.variant.no/#andre-goder-og-ytelser))
+
+## Om Variant
+
+![Christian, Jacob og Maciek hos Variant](/images/utvikler-jacob.png)
+Variant er menneskene som jobber her. Vi har kommet sammen for å være med å skape et konsept og en filosofi vi selv tror på. Hverdagen blir mer spennende av å ha et fellesskap som engasjeres av utvikling og hvordan det påvirker hverdagen. Forretningsmodell vår er enkel: Vi tilbyr flinke folk.
+
+Da er det helt selvsagt at vi må investere i kunnskap. Læreglede kommer i mange fasonger og vi prøver å tilrettelegge for at vi som varianter skal lære på den måten vi ønsker. Noen av oss driver podcasts ([her](http://bartjs.io/tag/podcast-episode/) og [her](https://kortslutning.fun/)), noen lager [kodevideoer](https://youtube.com/kodesnutt), noen underviser på NTNU. I tillegg samles vi hver måned til det vi [kaller en variantdag](https://blog.variant.no/tagged/variantdag); vår egen innedag for faglig påfyll. Og, ikke minst, for å møtes.
```

Comment:
Dette er jo et veldig fint avsnitt og viser at det er mange dyktige folk i Variant, men det kan også virke litt skremmende hvis du ikke gjør "noe ekstra" fra før av. Foreslår å legge til at læreglede ikke kun handler om å dypdykke i det en jobber med, men også dele andre aspekt - f.eks da Hilde holdt foredrag om hvordan psykologi spiller en rolle innen design.

```suggestion
Da er det helt selvsagt at vi må investere i kunnskap. Læreglede kommer i mange fasonger og vi prøver å tilrettelegge for at vi som varianter skal lære på den måten vi ønsker. Noen av oss driver podcasts ([her](http://bartjs.io/tag/podcast-episode/) og [her](https://kortslutning.fun/)), noen lager [kodevideoer](https://youtube.com/kodesnutt), noen underviser på NTNU og andre deler noe interessant de har lært den siste tiden gjennom uformelle lærekvelder eller foredrag. I tillegg samles vi hver måned til det vi [kaller en variantdag](https://blog.variant.no/tagged/variantdag); vår egen innedag for faglig påfyll. Og, ikke minst, for å møtes.
```

Reply:
Veldig bra innspill, helt enig @sarahse 🙌 😄  Jeg la også til at "noen skriver bloggposts" i starten av setningen og utvidet ditt avsnitt med at man rett og slett også bare kan dele ting på vår Slack (for å senke skuldrene enda mer).